src/shared/meson.build

```diff
@@ -146,7 +146,7 @@ shared_sources = files(
         'portable-util.c',
         'pretty-print.c',
         'capsule-util.c',
-        'ptyfwd.c',
+        'ptyfwd-util.c',
```

Comment:
i am not sure "-util" suffix for this really makes sense. The way i see it "-util" suffix adds some utility calls for some other subsystem. But "ptyfwd" is not really an "other" subsystem, it is geneuinely an inventation of ours here...

i'd go the other way, and leave ptyfwd the way it is and then introduce the new tool as ptyfwd-tool.c or so? we started this logic in loop/loop-tool.c, mount/mount-tool.c and vpick/vpick-too.c, and I think that makes sense and should be continued for tools that are pretty much just cmdline interfaces for some specific functionality of ours.